components/alert.css

```diff
@@ -0,0 +1,41 @@
+.alert {
```

Comment:
Could set a min-width here so that it doesn't get too small on smaller screens. Something like 10rems. And use media query to set different widths for different sizes